common/djangoapps/third_party_auth/saml.py

```diff
@@ -231,10 +231,10 @@ def odata_company_id(self):
     def odata_client_id(self):
         return self.conf['odata_client_id']
 
-    def missing_variables(self):
+    def invalid_configuration(self):
         """
         Check that we have all the details we need to properly retrieve rich data from the
-        SAP SuccessFactors OData API. If we don't, then we should log a warning indicating
+        SAP SuccessFactors BizX OData API. If we don't, then we should log a warning indicating
```

Comment:
What is `BizX`?

Reply:
BizX is the actual name of the SuccessFactors system -- so for example, people at SAP refer to the LMS as "BizX Learning", and if you want to provision a new user in the system you go into "BizX Provisioning", etc.

Reply:
Does that mean then that you should move the reference to "SuccessFactors" entirely, i.e. `SAP BizX OData API`?

Reply:
I guess it's up to us -- the whole thing is a branding mess at the moment due to the fact that these are acquired systems.  "SAP SuccessFactors" will likely become just "SAP" in another year or so, and "BizX" may or may not stick around.  Currently the full name of the LMS module is "SAP SuccessFactors BizX Learning".  The correct name of the OData API is "SAP SuccessFactors BizX OData API".

Reply:
This is a bit clearer in the Postman collection I've published in our private config repo.